docs/reference/configuration/private-pki.md

```diff
@@ -0,0 +1,160 @@
+
+Example scenarios include:
+- **Your domain cert is self-signed or private PKI**: Grafana would need the CA for SSO to work with Keycloak
+- **External dependencies use private PKI**: Velero, Loki (object storage) and potentially Grafana/Keycloak for databases, data sources, external identity providers
+
+This guide explains how to configure UDS Core components to recognize and trust your private CA certificates. Not every component requires this configuration — only those that make outbound TLS connections (for example, to identity providers, object storage, or other HTTPS endpoints).
+
+:::tip[Who should use this guide?]
+If your UDS Core environment connects to services using **self-signed certificates** or certificates issued by a **private CA**, you'll need to follow this configuration.
+If you only use certificates from public, trusted CAs (e.g., Let's Encrypt, DigiCert), you do **not** need these steps.
+:::
+
+:::caution[Security Consideration]
+Mounting additional volumes and certificates can introduce minimal security risks by expanding the attack surface. Only mount trusted CA certificates from verified sources and regularly audit your certificate configurations.
+:::
+
+## Prerequisites
+
+Before configuring private PKI, you'll need the following:
+
+1. The trusted CA bundle in PEM format that your certificates are signed by
+2. A ConfigMap containing the trusted CA bundle from (1), available in each namespace (a tool like [trust-manager](https://cert-manager.io/docs/trust/trust-manager/) can help automate this)
+
+:::note
+For the examples in this guide, we assume you have a ConfigMap named `private-ca` with a key `ca.pem` that contains your CA bundle.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: private-ca
+  namespace: <your-namespace>
+data:
+  ca.pem: |
+    -----BEGIN CERTIFICATE-----
+    MIIDeTCCAmGgAwIBAgIUQj...<snip>...hQ==
+    -----END CERTIFICATE-----
+```
+:::
+
+## Component Configuration
+
+### Authservice
+
+Authservice can be configured with an additional trusted CA bundle when UDS Core ingress gateways are deployed with private PKI.
+
+To configure, set `UDS_CA_CERT` as an environment variable with a Base64 encoded PEM formatted CA bundle that can be used to verify the certificates of the tenant gateway.
+
+See [trusted CA SSO doc](/reference/configuration/single-sign-on/trusted-ca) for complete Authservice configuration details.
+
+### Grafana
+
+Grafana may need to access external https services.  If these services use certificates signed by a private CA, Grafana needs to be configured to trust these certificates.
+
+Configure Grafana to trust private certificates by mounting and overriding the CA bundle via UDS Bundle overrides:
+
+```yaml
+values:
+  - path: extraConfigMapMounts
+    value:
+      - name: ca-certs
+        mountPath: /etc/ssl/certs/ca-certificates.crt # For Debian/Ubuntu images
```

Comment:
just a small nit: should this specify `(unicorn or upstream flavor)` to be explicit about when to use this mount path based on flavor?